pkg/config/datafileprojectconfig/config.go

```diff
@@ -48,6 +49,11 @@ type DatafileProjectConfig struct {
 	botFiltering         bool
 }
 
+// GetDatafile returns a JSON string representation of the environment's datafile
```

Comment:
returns a string representation, datafile is itself in JSON, don't need to add that.